### DISCUSSION TITLE
Renault Austral does not repeat albums

### DISCUSSION THREAD

**Discussion by marek1509** [MoleMan1024/audiowagon#154] [Q&A]
Witam u mnie w Renault Austral nie działa powtarzanie albumów i losowego odtwarzania utworów, jest ikona z prawej strony, po naciśnięciu jej pojawiają się ikony u góry powtarzania albumów i losowego odtwarzania, ale są nieaktywne takie przyciemnione.
Drugą sprawą mniej ważną jest ,że po akceptacji dysku usb długo trzeba czekać  ok 20 sekund na muzykę.Pozdrawiam z Polski
Wersja AUDIOWAGON 2.7.0

**Comment by MoleMan1024** [comment]
Hi, please keep all discussions in English only. Google Translate:

> Hello, in my Renault Austral, repeating albums and random playback of songs does not work, there is an icon on the right side, after pressing it, icons for repeating albums and random playback appear at the top, but they are inactive, so dim.
> 
> The second less important thing is that after accepting the USB drive, you have to wait a long time, about 20 seconds for the music. Greetings from Poland
> Version AUDIOWAGON 2.7.0

Regarding 1) somebody has already told me this once before via e-mail but did not get back to me when I asked further questions. Does this issue appear also with any other media app using the Renault Android system? Please try other media apps e.g. Spotify, Deezer, Radio or whatever and look for similar extra/sub-menus that can be opened by tapping these three dots and check if they are grayed out. If those are grayed out on other apps too, the issue is on Renault side.

For item 2) What you describe is not clear to me, please check if [the FAQ page](https://moleman1024.github.io/audiowagon/faq.html#why-does-it-take-so-long-when-starting-the-app) answers your question.

**Comment by marek1509** [comment]
Hello, thanks for the quick answer. I checked on SPOTIFY, you can select it and the random selection of songs works. In the second case, after entering the car, a screen appears with a check mark to accept the USB drive, after acceptance, it takes about 30 seconds before the music starts. It's not a tragedy, but I'm reporting it. that's how it is in Renault Austral.

**Reply by MoleMan1024** [reply]
For 1) please submit a log file as described [here](https://moleman1024.github.io/audiowagon/#how-do-i-report-an-issue).

2) For the 30 seconds wait time after accepting the USB permission, did you read the FAQ page I linked? Did you try to change the metadata indexing mode? If you have set it to "when USB connected" it will take some seconds for the indexer to look for new files always.

**Reply by MoleMan1024** [reply]
For 1) I have received the log files you sent by e-mail. Unfortunately everything looks normal in those, the custom actions for shuffle/repeat are there and look normal. Also these work normally in the Polestar 2 Android 10 emulator, so it is not a general Android 10 problem. So this problem must be specific to something that the Renault system does. Without having an emulator image or having a car available to me for further debugging or getting more detailed logs, I don't know what would cause this.

**Comment by marek1509** [comment]
Hello, thanks for your time and interest in the problem. I think that someday after Android 10 or Audiowagon updates it will start working.
Regards.

**Reply by MoleMan1024** [reply]
To be clear, there will not be any update from AudioWagon to address this, because I don't know how to analyze it as explained above. It is unlikely to be fixed randomly because I have no reason to touch this area of the code. 
An update of the Renault Android system to a higher version might help or it might not.

**Reply by marek1509** [reply]
Witam wysyłam zdjęcia z ekranu samochodu jeśli jesteś jeszcze zainteresowany.
![IMG_20241207_155359](https://github.com/user-attachments/assets/40eef0dd-7b1b-4581-bb32-72906143d15e)
![IMG_20241207_155409](https://github.com/user-attachments/assets/b15658c3-4201-45aa-8ae6-dee86ccfbd5f)

**Reply by marek1509** [reply]
Hello, I'm sending photos from the car screen if you are still interested.

**Reply by MoleMan1024** [reply]
If you can not click on the icons in your second image, then this is a bug in the Renault system, I can not influence this from my app.

**Reply by marek1509** [reply]
Thanks for the answer, I will try to find out if this problem is only with me or in all Renault Australs

**Reply by somehuman60** [reply]
> Thanks for the answer, I will try to find out if this problem is only with me or in all Renault Australs

The same on my Renault Megane E-Tech. Audiowagon 2.8.7.
